storage/flux/reader.go

```diff
@@ -401,6 +401,59 @@ const (
 	valueColIdx = 3
 )
 
+func determineTableColsForWindowAgg(tags models.Tags, typ flux.ColType, aggregate bool) ([]flux.ColMeta, [][]byte) {
```

Comment:
This is a seemingly small thing, but is there any benefit in truncating "Aggregate" to "Agg"? Could we fully expand it?

Reply:
To be honest, it's already such a long name that a few more characters won't hurt. There are similar functions with very similar names in this file already, so I didn't want to break from convention. But I do think renaming this function to something shorter would be much more readable.